.travis.yml

```diff
@@ -25,8 +25,9 @@ env:
     - RUST_CLIPPY_VERSION=0.0.211
     - EJB_RUST_BUILD_DIR="$TRAVIS_BUILD_DIR/exonum-java-binding-core/rust/"
     # force building instead of using from apt.
-    - ROCKSDB_BUILD=1
-    - SNAPPY_BUILD=1
+    # let's see what would happen
+    # - ROCKSDB_BUILD=1
```

Comment:
Please note that this is required on _some_ (older) Linux systems — it will most likely build fine on Linux this time.

Reply:
Trusty used by Travis is pretty old though, however, ROCKSDB_BUILD and SNAPPY_BUILD would be set only for linux system.